io.github.elograf.Elograf.metainfo.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.elograf.Elograf</id>
```

Comment:
this app-id is invalid. There is no "elograf" organisation or user on github.

Reply:
Fixed